src/Aspire.Hosting/Eventing/IDistributedApplicationEventing.cs

```diff
@@ -0,0 +1,30 @@
+namespace Aspire.Hosting.Eventing;
+
+/// <summary>
+/// Supports publishing and subscribing to events which are executed during the AppHost lifecycle.
+/// </summary>
+public interface IDistributedApplicationEventing
```

Comment:
`IDistributedApplicationEventHub`? ☺️

Reply:
I think the Event Hubs team will send us hate mail if we do that ;)

Reply:
That's too funny 😆